### ISSUE TITLE
Mesh object output error

### ISSUE THREAD

**Issue by eighty8serge** [JacquesLucke/animation_nodes#1956]
Blender 4.1.0 + AN 3.11
System: Win10 64

![1](https://github.com/JacquesLucke/animation_nodes/assets/151669582/9f9897ec-112d-42ce-91b6-d14d7017f429)

If input mesh uses _Load Custom Attribute_ checkbox, then error arise:

```
Traceback (most recent call last):
  File "E:\blender-4.1.0-beta+v41.2330e2564f69-windows.amd64-release\4.1\scripts\addons\animation_nodes\execution\main_execution_unit.py", line 38, in executeUnit
    exec(self.executeCodeObject, self.executionData, self.executionData)
  File "execution: 'Animation Nodes'", line 45, in <module>
  File "E:\blender-4.1.0-beta+v41.2330e2564f69-windows.amd64-release\4.1\scripts\addons\animation_nodes\nodes\mesh\mesh_object_output.py", line 175, in setMesh
    attributeOut.data.foreach_set("value", data.asMemoryView())
TypeError: couldn't access the py sequence
```

**Comment by OmarEmaraDev** [comment]
Does it work in Blender 4.2?

**Comment by eighty8serge** [comment]
Unfortunately, no, tried blender-4.2.0-alpha+main.fae16e74547f-windows.amd64-release

**Comment by OmarEmaraDev** [comment]
Fixed in fe45013fd022cd0ea50c2f24ac91322d2e719d13.

**Comment by eighty8serge** [comment]
There are still some issues after update:

```
Info: Registering operator class: 'InvokeFunction_0', bl_idname 'an.invoke_function_0' has been registered before, unregistering previous
E:\blender-4.2.0-alpha+main.fae16e74547f-windows.amd64-release\4.2\scripts\addons\animation_nodes\nodes\mesh\mesh_object_output.py:180: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
  attributeOut.data.foreach_set("value", data.asNumpyArray() != 0)
Error: Array length mismatch (expected 8, got -1)
TypeError: object of type 'bool' has no len()

Traceback (most recent call last):
  File "E:\blender-4.2.0-alpha+main.fae16e74547f-windows.amd64-release\4.2\scripts\addons\animation_nodes\execution\main_execution_unit.py", line 38, in executeUnit
    exec(self.executeCodeObject, self.executionData, self.executionData)
  File "execution: 'Animation Nodes'", line 45, in <module>
  File "E:\blender-4.2.0-alpha+main.fae16e74547f-windows.amd64-release\4.2\scripts\addons\animation_nodes\nodes\mesh\mesh_object_output.py", line 180, in setMesh
    attributeOut.data.foreach_set("value", data.asNumpyArray() != 0)
TypeError: couldn't access the py sequence
Error: Array length mismatch (expected 8, got -1)
TypeError: object of type 'bool' has no len()
```

**Comment by OmarEmaraDev** [comment]
Hopefully should be fixed in b892924b0d92d80babd55c1f74059c87c68c6278.
